### ISSUE TITLE
Segmentation with transparent background

### ISSUE THREAD

**Issue by molnarszilard** [DIYer22/bpycv#47]
Hi, 
I am trying to do a semantic segmentation for a dataset containing plants.
My models contain a lot of leaves, which are basically built using support squares, with an image of a leaf on them.
In these images the background is transparent.
In the RGB render, the leaves appear normal, the transparency works - I can only see the leaves, not their supports.
However, in the depth and semantic windows, the leaves appear square (I tried .obj and .fbx formats).
Is there a setting that I can use, so that, the semantic mask is created using only the visible parts, without their transparent support square? Or do I have to fix the plant models first? (In Nvidia Omniverse the segmentation is correct for these models.)

Thanks.
![0](https://user-images.githubusercontent.com/60471017/222251055-a7e23d9e-9962-4905-9efe-9e5afaa5714d.jpg)

**Comment by DIYer22** [comment]
Sorry, bpycv's instance mask currently does not support transparency.
If you still continue to use bpycv, please consider fixing the plant models.

**Comment by beyse** [comment]
@DIYer22 what needs to be done to support transparency? 

@molnarszilard did you found a solution or work around?

**Comment by zsy950116** [comment]
您好，您的邮件我已经接收。

**Comment by beyse** [comment]
好的，酷

**Comment by molnarszilard** [comment]
No, unfortunately, I did not find a solution (although I did not have much time to deal with this problem). Maybe a clue could be a different Blender version (not with bpycv, but just in the bare Blender these squares behave differently depending on the different Blender versions - but at this moment I do not remember exactly the version number)
